CoreFoundation/Base.subproj/SwiftRuntime/CoreFoundation.h

```diff
@@ -85,19 +85,19 @@
 #include <CoreFoundation/CFUUID.h>
 #include <CoreFoundation/CFUtilities.h>
 #include <CoreFoundation/CFBundle.h>
+
+#if (TARGET_OS_MAC && !(TARGET_OS_EMBEDDED || TARGET_OS_IPHONE)) || (TARGET_OS_EMBEDDED || TARGET_OS_IPHONE) || TARGET_OS_WIN32
```

Comment:
? That is not the availability of those headers.

Reply:
That was copied from `Base.subproj/CoreFoundation.h`.  Perhaps that is also incorrect?

Reply:
Per offline conversation: just don't include the conditional here.